build/configs/rtl8730e/loadable_ext_ddr/defconfig

```diff
@@ -325,8 +325,8 @@ CONFIG_SECOND_FLASH_START_ADDR=0x0A000000
 CONFIG_RTL8730E_BOARD_REVISION=7
 CONFIG_RAM_DDR=y
 # CONFIG_RAM_PSRAM is not set
-CONFIG_USERFS_MNTPT="/mnt0"
-CONFIG_USERFS_EXT_MNTPT="/mnt"
+CONFIG_USERFS_MNTPT="/mnt"
```

Comment:
If this PR is official, should this configuration be defined in Kconfig? 
Also, does this macro need to be applied to other config?

Reply:
Now, this is for testing. When we merge your PR for internal flash use, we will update this PR as well with all necessary defconfigs changes.